app/views/books/_form.html.erb

```diff
@@ -1,7 +1,13 @@
-<%= simple_form_for(@book, html: { class: "form-inline" }) do |form| %>
-  <%= form.input(:name, input_html: { class: "m-1" }) %>
-  <%= form.input(:isbn, label: "ISBN", input_html: { class: "m-1" }) %>
-  <%= form.input(:featured, label: "Featured?", as: :boolean, input_html: { class: "m-1" }) %>
+<%= simple_form_for(@book) do |form| %>
+  <%= form.input(:name) %>
+  <%= form.input(:isbn, label: "ISBN") %>
+  <%= form.input(:featured, label: "Featured?", as: :boolean) %>
   <%= form.drag_and_drop_file_field(:cover) %>
   <%= form.submit %>
 <% end %>
+
+<style>
+  #book_cover {
+    z-index: -5;
+  }
+</style>
```

Comment:
Is this still required? I remember we had talked about this and it was related to the form when it was being displayed horizontally.

Reply:
Nope! Removed